README.md

```diff
@@ -20,58 +20,43 @@ The main objective of this challenge is to test your skills in Front-end Develop
 
 ## The Challenge
 
-The [following layout](layout.jpg) need to be developed.
+### About layout
 
-<img src="layout.jpg" width="800">
+The [following Figma layout needs to be developed](https://www.figma.com/file/ED8oK1CyH2BD5BjSAcy9vF/Teste-Front-end?node-id=505%3A2613).
 
-It's important to say that it is just a prototype, the idea is to check your ability to propose improvements, features and contribute to the product UI 😄. Feel free to make improvements like new functionalities, animations, visual optimizations, etc.
+<img src="layout-jsm.png" width="800">
+
+It's important to say that it is just a prototype, the idea is to check your ability to propose improvements, features and contribute to the product UI 😄.
 
 The logo is the same as the one on the repository.
```

Comment:
**suggestion (non-blocking):** my friend, what do you think of the following structure:

```suggestion
The logo is the same as the one in the repository.

It's important to say that it is just a prototype! We would also like to see your ability to propose improvements, features and new contributions to the product's UI 😄!
```